### PR TITLE
Barrier fix, Content log Fixes

### PR DESCRIPTION
**Pull request by MegadronA03**
### Summary:

Fixed all content log errors.
Barriers now no more invisible.
Entity rendering changes.

---------
### Screenshots:

Screenshot_20210716-153851_Minecraft

### CODE REVIEW

**Comment by jebbyk** [comment]
@MegadronA03 Oh, nice, also you can add your self to developers list in mainfest.json and loading messages

**Comment by MegadronA03** [comment]
Added

File: shaders/glsl/animations.glsl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#define WIND_SPEED 8.0
```
Comment (jebbyk) [inline shaders/glsl/animations.glsl]: @MegadronA03  are u sure this file in a correct directory? I think it should be in /glsl/options/preset

**Review by jebbyk — COMMENTED** [review]
I suggest you to move  five last commits to separte PR. I want to release 0.13.2 today. but I have no time to test all the stuff you've added. Please leave only content log and barier fix here :-) thanx.

**Pull request closed without merging**